pkg/cmd/convert.go

```diff
@@ -62,15 +69,39 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
```

Comment:
I think that these two variable should stay declared inside the loop as they were before. Otherwise you are extending the scope and a life time of them which can be error prone